client/src/templates/Challenges/components/prism-formatted.tsx

```diff
@@ -1,26 +1,26 @@
 import React, { Component } from 'react';
 import Prism from 'prismjs';
-import PropTypes from 'prop-types';
 
-const propTypes = {
-  className: PropTypes.string,
-  text: PropTypes.string.isRequired
-};
+interface PrismFormattedProps {
+  className?: string;
+  text: string;
+}
 
-class PrismFormatted extends Component {
-  componentDidMount() {
+class PrismFormatted extends Component<PrismFormattedProps> {
+  instructionsRef: React.RefObject<HTMLInputElement>;
+  componentDidMount(): void {
```

Comment:
```suggestion
  displayName = 'PrismFormatted';
  instructionsRef: React.RefObject<HTMLInputElement>;
  componentDidMount(): void {
```

Reply:
From what I’ve seen a static declaration is usually used. I’ll make that change